mmv1/templates/terraform/examples/vertex_ai_featureonlinestore_with_optimized.tf.erb

```diff
@@ -1,5 +1,5 @@
 resource "google_vertex_ai_feature_online_store" "<%= ctx[:primary_resource_id] %>" {
-  provider = google-beta
+  provider = google
```

Comment:
nit: This is technically redundant as we'll use the default `google` provider anyways. Feel free to remove if you'd like in a followup, I'm gonna merge regardless, as it's pretty minor.